source/music_album.rb

```diff
@@ -0,0 +1,16 @@
+require '../source/item'
+class MusicAlbum < Item
+  attr_accessor :title, :on_spotify, :archived
+
+  def initialize(title, on_spotify)
+    super(title)
+    @on_spotify = on_spotify
+    @archived = false
+  end
+
+  private
+
+  def can_be_archived?
+    @archived = super && @on_spotify
+  end
```

Comment:
-  Please there is no title as an argument but rather a date based on the diagram
-  superclass accepts date as an argument
- According to the requirement, the archived property is only updated when the "move_to_archive" method is called and the condition is true. can_be_archived? should not modify it but should only return true or false